Gemfile

```diff
@@ -32,6 +32,7 @@ group :default do
 
   gem 'puma'
   gem 'sanger_barcode_format', github: 'sanger/sanger_barcode_format', branch: 'development'
+  gem 'sprockets-rails'
```

Comment:
No longer a default dependency.

Moving limber on to vite may be a touch complicated by invline_svg which tries to hook into the Rails asset pipeline. The latest versions are configurable though, so I'm sure we'll manage.